src/java.base/share/classes/java/io/File.java

```diff
@@ -2117,6 +2117,11 @@ static File generateFile(String prefix, String suffix, File dir)
      * made the name of the new file will be generated by concatenating the
      * prefix, five or more internally-generated characters, and the suffix.
      *
+     * <p> If a file having the generated name cannot be created by the
```

Comment:
I think it might be clear to say "with the generated name".

Reply:
So changed in 64440ec.